frontend/integration-tests/views/kubevirt/vm.actions.view.ts

```diff
@@ -0,0 +1,33 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { rowForName, actionsDropdown, actionsDropdownMenu } from '../crud.view';
+
+
+/**
+ * Performs action for VM via list view kebab menu.
+ */
+export const listViewAction = (name: string) => (action: string) => rowForName(name).$$('.co-kebab__button').first().click()
```

Comment:
would it be possible to extract common functionality from listViewAction and detailViewAction? And also, extracting the kebap into a variable?

Reply:
Done. Couldn't find a way to extract the kebab element, for some reason when I had 
`export const $('.co-kebab__dropdown');`
before the action function, the element was undefined.

Reply:
coulnd't we just pass a function, something like `getActionsDropdown()`?

Reply:
As far as I know, we can't. Such function would have to be implemented by ElementFinder, which is what rowForName(name) returns.
But maybe just defining the string could be OK?
Like:
`const listKebabDropdown = '.co-kebab__dropdown';`
`rowForName(name).$(listKebabDropdown)...`

Reply:
would this work or am I missing something?
```js
listViewAction = (name) => {
    getActionsDropdown = () => rowForName(name).$$('.co-kebab__button').first();
    getActionsDropdownMenu = () => rowForName(name).$(listViewKebabDropdown);
    return selectDropdownItem(getActionsDropdown, getActionsDropdownMenu);
}

selectDropdownItem  = (getActionsDropdown, getActionsDropdownMenu) => (action) =>
.
. common functionality (calls getActionsDropdown and getActionsDropdownMenu)
.
```

and `detailViewAction` implemented with the same mechanism

Reply:
Sorry for the delay, I'm volunteering on QE Camp. Yes, this approach will work.

Reply:
Done.